src/index.js

```diff
@@ -14,16 +14,16 @@ export {
   StyleUtils,
   version,
 };
-export { default as Button } from './button';
 export { default as Banner } from './banner';
+export { default as Button } from './button';
 export { default as Checkbox } from './checkbox';
-export { default as Input } from './input';
 export { default as Label } from './label';
 export { default as Modal } from './modal';
 export { default as Overlay } from './overlay';
 export { default as Portal } from './portal';
 export { default as Spinner } from './spinner';
 export { DatePicker, TimePicker } from './datetime';
+export { Input, TextArea } from './input';
```

Comment:
How do you feel about renaming this directory as `text` instead?

Reply:
Not every input is for the `text` type. Could be for `number`, 'password', 'tel' etc.

Reply:
My understanding was that `Input` is meant to be used only for `type="text"`. If that's the case, this directory contains text-related components, so that's where I'm coming from.